examples/mlp.py

```diff
@@ -13,16 +13,35 @@
 # The following import installs custom tracebacks for inference errors
 from myia.debug import traceback  # noqa
 
+from myia.abstract import from_value 
+from myia.compile.backends import load_backend
+
+from myia.api import to_device
+
+from myia.dtype import Float
+from myia.prim.py_implementations import scalar_cast
 
 ###########
 # Options #
 ###########
 
 
 dtype = 'float32'
+
+backend = 'pytorch'
+#backend = 'nnvm'
+#backend = 'relay'
+
 device_type = 'cpu'
 # device_type = 'cuda'  # Uncomment to run on the gpu
 
+if backend == 'pytorch':
+    backend_options = {'device': device_type}
+elif backend == 'nnvm':
+    backend_options = {'target': device_type, 'device_id': 0}
+elif backend == 'relay':
+    backend_options = {'target': device_type, 'device_id': 0}
```

Comment:
I'd put these in a dictionary, e.g. `backend_options = {'pytorch': {'device': device_type}, ...}`, and index it with `backend` in the decorator. It's a bit more flexible and the error will be a little nicer if there's a typo in the backend name.